<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d3dRenderEngineClass</class>
 <widget class="QMainWindow" name="d3dRenderEngineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>d3dRenderEngine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="RenderWidget" name="render_widget" native="true">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Scenes</string>
    </property>
    <addaction name="actionHillWaves"/>
    <addaction name="actionMirror"/>
    <addaction name="actionTessellation"/>
    <addaction name="actionAltar"/>
    <addaction name="actionCubeMap"/>
    <addaction name="actionNor_DispMap"/>
    <addaction name="actionTerrain"/>
    <addaction name="actionParticle"/>
    <addaction name="actionShadow"/>
    <addaction name="actionSSAO"/>
    <addaction name="actionAnimation"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHillWaves">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HillWaves</string>
   </property>
  </action>
  <action name="actionMirror">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
  </action>
  <action name="actionAltar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Altar</string>
   </property>
  </action>
  <action name="actionNor_DispMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NorDispMap</string>
   </property>
  </action>
  <action name="actionTerrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Terrain</string>
   </property>
  </action>
  <action name="actionParticle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Particle</string>
   </property>
  </action>
  <action name="actionTessellation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tessellation</string>
   </property>
  </action>
  <action name="actionShadow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shadow</string>
   </property>
  </action>
  <action name="actionSSAO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SSAO</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Animation</string>
   </property>
  </action>
  <action name="actionCubeMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CubeMap</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QWidget</extends>
   <header>RenderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="d3drenderengine.qrc"/>
 </resources>
 <connections/>
</ui>
